proposals/0107-unsaferawpointer.md

```diff
@@ -1477,34 +1477,36 @@ struct UnsafeMutableRawPointer : Strideable, Hashable, _Pointer {
   func bindMemory<T>(to: T.Type, capacity: Int) -> UnsafeMutablePointer<T>
   func assumingMemoryBound<T>(to: T.Type) -> UnsafeMutablePointer<T>
 
-  func initialize<T>(_: T.Type, with: T, count: Int = 1)
+  func initialize<T>(_: T.Type, to: T, count: Int = 1)
     -> UnsafeMutablePointer<T>
-  func initialize<T>(toContiguous: T.Type, atIndex: Int, with: T)
+  func initialize<T>(contiguous: T.Type, at: Int, to: T)
```

Comment:
I read it as "initialize contiguous `String` at `position` to `"hello"`"—that is, assume the pointer points to contiguous `String`s, and initialize the `String` at `position` to `"hello"`.

If you don't think that reads well, I wonder if we can combine both the `at` and `count` variants into one call:

```swift
func initialize<T>(_: T.Type, at: Int = 0, count: Int = 1, to value: T) -> UnsafeMutablePointer<T>

address.initialize(String.self, to: "hello")
address.initialize(String.self, at: 3, to: "hello")
address.initialize(String.self, count: 3, to: "hello")
address.initialize(String.self, at: 3, count: 3, to: "hello") 
```

If the bare type doesn't read well, it could have an `as` label.